### PR TITLE
DHFPROD-7187: Change Flow Run Settings In Run Tile

### PR DESCRIPTION
**Pull request by MSBrignoni**
### Description
Change Flow Run Settings In Run Tile

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests
- [x] Added to Release Wiki

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/components/flows/flows.tsx [hunk 1]
```diff
@@ -716,27 +807,29 @@ const Flows: React.FC<Props> = (props) => {
...
-        await props.runFlowSteps(runningFlow, stepNumbers, formData)
-          .then(resp => {
-            setShowUploadError(true);
-            setFileList([]);
-            setSelectedStepOptions({});
-            setSelectedStepDetails([{stepName: "", stepNumber: -1, stepDefinitionType: "", isChecked: false}]);
-            setRunFlowClicked(false);
-          });
-      }
+      // if (!runFlowClicked) {
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/src/components/flows/flows.tsx:810]: These commented lines are part of another story?

Comment (MSBrignoni) [reply marklogic-data-hub-central/ui/src/components/flows/flows.tsx:810]: Yes @aniamy , Is part of the previous story, right now I'm not using this lines but could be necessary in the next pr, if not, at the end i will erase it!.

**Review by aniamy — DISMISSED** [review]

**Review by yubiselmarklogic — APPROVED** [review]

**Comment by briantang** [comment]
Looks good but let's wait for @lucianaabc-ml 's PR to go in to see if it fixes your test failures.

**Comment by MSBrignoni** [comment]
> Looks good but let's wait for @lucianaabc-ml 's PR to go in to see if it fixes your test failures.

Done and rebased!

**Review by briantang — COMMENTED** [review]
@MSBrignoni Looks like there's still a couple failures to be looked at here. Let me know if you need help with them.

**Review by lucianaabc-ml — APPROVED** [review]

**Review by briantang — APPROVED** [review]

**Pull request merged**